decisiontree/models.py

```diff
@@ -198,6 +199,27 @@ def __str__(self):
         return u"%s : %s --> %s" % (self.current_state, self.answer, self.next_state)
 
 
+class SessionQuerySet(models.query.QuerySet):
+
+    def open(self):
+        return self.exclude(Q(state=None) | Q(canceled=True))
```

Comment:
This one could be done without a Q(), right? `self.exclude(state=None).exclude(canceled=True)`